dockers/README.md

```diff
@@ -0,0 +1,227 @@
+
+### <a name="scanning"></a> Image scanning
+
+
+All of the images that we build are scanned for critical vulnerabilities on every pull request. For this we use a github-action that leverages [trivy](https://github.com/aquasecurity/trivy) for scanning. If you build a new image please add it to the action [here](../.github/workflows/trivy.yml).
+
+### <a name="semantic"></a> Semantic tagging
+
+
+We recommend against using rolling tags like `master` or `latest` when building images. Rolling tags make it hard to track down versions of images since the underlying image hash and content could be different across the same tags. Instead we ask that you use a semantic tag that follows the convention below:
+
+##### `us.gcr.io/broad-gotc-prod/samtools:<image-version>-<samtools-version>-<unix-timestamp>` 
+
+This example is for an image we use containing `samtools`. The 'image-version' in this case is the traditional `major.minor.patch` version of the image being built, which is updated when changes to the image (underlying OS, system level packages, etc.) unrelated to `samtools` are made. The 'samtools-version' here correlates with the specific version of `samtools` being used, having this information in the tag makes it easy for users to identify and not have to track down.
+
+### <a name="process"></a> Proper process reaping
+
+
+Classic init systems like systemd are used to reap orphaned, zombie processes. Typically these orphaned processes are reattached to the process at PID 1 which will reap them when they die. In a container this responsibility falls to process at PID 1 which is by default `/bin/sh`...this obviously will not handle process reaping. Because of this you run the risk of expending excess memory or resources within your container. A simple solution to this is to use `tini` in all of our images, a lengthy explanation of what this package does can be found [here](https://github.com/krallin/tini/issues/8).
+
+Luckily `tini` is available natively through APK so all you have to do is install it and set it as the default entrypoint!
+
+##### :eyes: Example
+```dockerfile
+
+FROM alpine:3.9
+
+RUN set eux; 
+        apk add --no-cache \
+            tini
+
+ENTRYPOINT ["/sbin/tini" , "--"]
+```
+
+## <a name="build"></a> Build Scripts and README
+
+To make life easier when building and pushing our images we like to have an easy-to-use `docker_build.sh` that sits next to each Dockerfile. These scripts should have configurable inputs for the version of tools (samtools, picard, zcall, etc.) being used in the image. Additionally, we like to keep a record of the versions built and being used by writing the images to the accompanying `docker_versions.tsv`.
```

Comment:
Is the `docker_versions.tsv` file still being automatically updated? If so, you might mention that here